modules/API/pages/gsql-endpoints.adoc

```diff
@@ -6357,11 +6365,11 @@ Sample Response::
 
 '''
 
-=== download all files
+=== download all user-defined function files
```

Comment:
"All" files seems to be an exaggeration.
As far as I can tell, this is only for the user-defined function files. If is is actually for a wide range of files, please explain.